test/gtfs/cache_file_test.exs

```diff
@@ -12,11 +13,11 @@ defmodule Gtfs.CacheFileTest do
 
   describe "save_gtfs/1 and load_gtfs/1" do
     test "saves to the default file, returns {:ok, term} for a map" do
-      data = %Gtfs{
+      data = %Data{
```

Comment:
If you add default values in the `Data` struct, you can avoid having to change these each time you add a new field.

Reply:
But then you lose the compile-time protection that you remembered all the fields in the real version.